test/over_react/component_declaration/flux_component_test/stateful/basic.dart

```diff
@@ -0,0 +1,21 @@
+part of over_react.component_declaration.flux_component_test;
```

Comment:
The rest of these files are just stateful versions of the existing flux test components